toolkit/tools/scheduler/scheduler.go

```diff
@@ -71,8 +71,9 @@ var (
 
 	ignoredPackages = app.Flag("ignored-packages", "Space separated list of specs ignoring rebuilds if their dependencies have been updated. Will still build if all of the spec's RPMs have not been built.").String()
 
-	pkgsToBuild   = app.Flag("packages", "Space separated list of top-level packages that should be built. Omit this argument to build all packages.").String()
-	pkgsToRebuild = app.Flag("rebuild-packages", "Space separated list of base package names packages that should be rebuilt.").String()
+	pkgsToBuild     = app.Flag("packages", "Space separated list of top-level packages that should be built. Omit this argument to build all packages.").String()
+	pkgsToRebuild   = app.Flag("rebuild-packages", "Space separated list of base package names packages that should be rebuilt.").String()
+	rpmHydratedTest = app.Flag("rpm-hydrated-test", "Is this just a TestRPM build with all RPMs hydrated?").String()
```

Comment:
This should just be a `.Bool()` flag I think, we don't encode anything beyond y/n here. See `$(RUN_CHECK)` and `$(STOP_ON_PKG_FAIL)` for examples of how to pass those in.